cmake/detray-config.cmake.in

```diff
@@ -32,7 +40,7 @@ set_and_check( detray_CMAKE_DIR "@PACKAGE_CMAKE_INSTALL_CMAKEDIR@" )
 
 # Print a standard information message about the package being found.
 include( FindPackageHandleStandardArgs )
-find_package_handle_standard_args( algebra-plugins REQUIRED_VARS
```

Comment:
Damn it , I caught the unimportant one :D

Reply:
:stuck_out_tongue: